stf/feature_engineering/feature_free_days.py

```diff
@@ -6,11 +6,13 @@
 
```

Comment:
Wouldn't it make more sense to call this module just `create_holiday_functions` or `holiday_function`. Perhaps it nice to be a bit more consistent with naming.